cpp/src/barretenberg/proof_system/flavor/flavor.hpp

```diff
@@ -275,23 +275,6 @@ template <class FF, size_t ExtendedLength, std::size_t Length = 2> static conste
     }
 }
 
-/**
```

Comment:
Note: this was leftover debg code that accidentally snuck in at some point. I replaced this method with `instantiate_barycentric_utils` which is the one being used.